app/models/partners/child.rb

```diff
@@ -11,7 +11,8 @@
 #  first_name           :string
 #  gender               :string
 #  health_insurance     :jsonb
-#  item_needed_diaperid :integer
+#  item_needed_diaperid :integer          # DEPRECATED
```

Comment:
This line is going to be blown away the next time `db:migrate` is run, FYI.

Reply:
addressed 8a823fe37654a82362d5d80014238e8fd6ec48d2